apps/src/lab2/views/components/editor/CodeEditor.tsx

```diff
@@ -33,13 +38,46 @@ const CodeEditor: React.FunctionComponent<CodeEditorProps> = ({
   const [editorView, setEditorView] = useState<EditorView | null>(null);
   const channelId = useAppSelector(state => state.lab.channel?.id);
   const isReadOnly = useAppSelector(isReadOnlyWorkspace);
+  const editorFontSizeKey = useAppSelector(
+    state => state.codebridgeWorkspace.editorFontSizeKey
+  );
+  const {signInState} = useAppSelector(state => state.currentUser);
+  const {levelProperties} = useCodebridgeContext();
```

Comment:
We can't rely on codebridge context here because this is a lab2 component. Maybe `appName` should be passed in as a prop?

Reply:
Similarly we probably shouldn't be referencing a codebridge-specific redux. Maybe that's a reason to make this setting in a lab2 redux? Or we could argue that this component will only ever be used by codebridge and move it.

Reply:
I initially thought about adding the stting in a lab2redux ([SIack thread](https://codedotorg.slack.com/archives/C03DBDN67B7/p1742934579582809)), and `aichat` doesn't currently have plans to increase font size. However, I do see that `CodeEditor` was recently added to `MusicLabView` by https://github.com/code-dot-org/code-dot-org/pull/64649 so I think moving the setting to lab2 may make sense.

Reply:
Moved redux state to new `lab2ViewRedux` - https://github.com/code-dot-org/code-dot-org/pull/64822/commits/566f2ffda55c0ebf328d749d52c7b300bfe3f5ce cc: @sanchitmalhotra126